components/Button/__test__/button.spec.ts

```diff
@@ -155,6 +155,31 @@ describe('Test: KButton', () => {
 		expect(host.innerHTML).matchSnapshot();
 	});
 
+	test('props: button size', async () => {
```

Comment:
Unit tests should be as comprehensive as possible.
Cover all three sizes.

Reply:
OK👌